openslides_backend/action/actions/mediafile/upload.py

```diff
@@ -4,6 +4,7 @@
 from time import time
 from typing import Any, Dict, List, TypedDict
 
+import magic
```

Comment:
Proposal: Personally I don't like the import name **magic**, because this name doesn't help anyone to understand, what it is for. If you agree, choose a good alias.
Could be something like `import magic as python_magic ` to identify the package (easier looking for documentation) or even more verbose.

Reply:
Done